eventhubs/client/azure-eventhubs/src/main/java/com/azure/eventhubs/EventHubClient.java

```diff
@@ -5,7 +5,9 @@
 
 import com.azure.core.amqp.AmqpConnection;
 import com.azure.core.amqp.exception.AmqpException;
+import com.azure.core.implementation.util.ImplUtils;
```

Comment:
Same thought as elsewhere;  do you want to bind the client directly to AMQP, or would it make sense to use an intermediate abstraction for things that are protocol/transport-aware?